unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/Tests/RPCSocialApiBridgeShould.cs

```diff
@@ -0,0 +1,234 @@
+using Cysharp.Threading.Tasks;
+using Cysharp.Threading.Tasks.Linq;
+using Decentraland.Social.Friendships;
+using MainScripts.DCL.Controllers.FriendsController;
+using NSubstitute;
+using NUnit.Framework;
+using rpc_csharp;
+using rpc_csharp.transport;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.Remoting.Contexts;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace DCL.Social.Friends
+{
+    class MockSocialServerContext
```

Comment:
We can use NSubstitute and avoid this class.

Reply:
The context helps with mocking, but changed the server for NSubstitute